addons/website/static/src/js/widgets/website_chart.js

```diff
@@ -0,0 +1,167 @@
+            chartData.options.scale = {
+                ticks: {
+                    beginAtZero: true,
+                }
+            };
+        } else if (['pie', 'doughnut'].includes(this.el.dataset.type)) {
+            chartData.options.tooltips.callbacks = {
+                label: (tooltipItem, data) => {
+                    const label = data.datasets[tooltipItem.datasetIndex].label;
+                    const secondLabel = data.labels[tooltipItem.index];
+                    let final = label;
+                    if (label) {
+                        if (secondLabel) {
+                            final = label + ' - ' + secondLabel;
+                        }
+                    } else if (secondLabel) {
+                        final = secondLabel;
+                    }
+                    return final + ':' + data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index];
+                },
+            };
+        } else {
+            chartData.options.scales = {
+                xAxes: [{
+                    stacked: this.el.dataset.stacked === 'true',
+                    ticks: {
+                        beginAtZero: true
+                    },
+                }],
+                yAxes: [{
+                    stacked: this.el.dataset.stacked === 'true',
+                    ticks: {
+                        beginAtZero: true
+                    },
+                }],
+            };
+        }
+
+        this.chart = new window.Chart(this.el, chartData);
+        return this._super.apply(this, arguments);
+    },
+    /**
+     *  Discard all library changes to reset the state of the Html.
+     */
+    destroy: function () {
+        const canvasEl = document.createElement('canvas');
+        for (const [key, value] of Object.entries(this.el.dataset)) {
+            canvasEl.dataset[key] = value;
+        }
+        this.$el.parent().empty().append($(canvasEl));
+        this._super.apply(this, arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {*} color
+     */
+    _convertToCssColor: function (color) {
+        if (!color) {
+            return 'transparent';
+        }
+        return this.style.getPropertyValue(`--${color}`).trim() || color;
+    },
+});
+
+publicWidget.registry.chart = publicWidget.Widget.extend({
+    selector: '.s_chart',
+    disabledInEditableMode: false,
+    /**
+     * @override
+     */
+    init: function () {
+        this._super.apply(this, arguments);
+    },
+    /**
+     * @override
+     */
+    start: function () {
+        const target = this.el.querySelector('canvas');
+        this._super.apply(this, arguments).then(() => {
+            const chart = new WebsiteChart(this);
+            chart.attachTo($(target));
+        });
+    }
+});
```

Comment:
Why ? Why not directly adding the correct selector on the `WebsiteChart` widget ?